.travis.yml

```diff
@@ -1,2 +1,10 @@
 language: ruby
+sudo: required
+services:
+  - rabbitmq
+before_script:
```

Comment:
You're right, I had the wrong understanding of travis' build stages...

I also mis-read the scripts themselves, because it looked like the end to end tests we're being executed before the unit specs. It looks like we're aligned now and the intentions are clear.